src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -169,3 +199,28 @@ export default class StickyToolbarView extends ToolbarView {
  * @observable
  * @member {Boolean} ui.stickyToobar.StickyToolbarViewModel#isSticky
  */
+
+/**
+ * Set `true` if the sticky toolbar reached the bottom edge of the
+ * {@link ui.stickyToolbar.StickyToolbarView#limiterElement}.
+ *
+ * @readonly
+ * @observable
+ * @member {Boolean} ui.stickyToobar.StickyToolbarViewModel#isStickyToLimiterBottom
```

Comment:
I feel that those should be made private. Especially that once we'll merge VMs into Vs, we shouldn't expose them too much. It will be natural to make them private (or protected) just like you did with https://github.com/ckeditor/ckeditor5-ui-default/pull/75/files#diff-3707d46f1d7a81c5face2db6d91da6efR89.